hack/dind-cluster.sh

```diff
@@ -104,9 +104,15 @@ function check-selinux() {
   fi
 }
 
+DIND_IMAGE="openshift/dind"
 IMAGE_REGISTRY="${OPENSHIFT_TEST_IMAGE_REGISTRY:-}"
+if [[ -n "${IMAGE_REGISTRY}" ]]; then
+  DIND_IMAGE="${IMAGE_REGISTRY}/${DIND_IMAGE}"
+fi
 IMAGE_TAG="${OPENSHIFT_TEST_IMAGE_TAG:-}"
-DIND_IMAGE="${IMAGE_REGISTRY}openshift/dind${IMAGE_TAG}"
+if [[ -n "${IMAGE_TAG}" ]]; then
+  DIND_IMAGE="${DIND_IMAGE}:${IMAGE_TAG}"
+fi
```

Comment:
This script got a lot of config options, either documenting under 'OpenShift Configuration' section or moving config params to hack/dind-config.sh will be helpful.